tvb_epilepsy/scripts/hypothesis_scripts.py

```diff
@@ -1,23 +1,125 @@
 
-from tvb_epilepsy.base.constants import EIGENVECTORS_NUMBER_SELECTION, WEIGHTED_EIGENVECTOR_SUM
-from tvb_epilepsy.base.utils import initialize_logger
+import os
+
+import numpy as np
+
+from tvb_epilepsy.base.constants.configurations import DATA_CUSTOM, FOLDER_RES, FOLDER_FIGURES
+from tvb_epilepsy.base.constants.module_constants import TVB, DATA_MODE, EIGENVECTORS_NUMBER_SELECTION, \
+                                                                                                WEIGHTED_EIGENVECTOR_SUM
+from tvb_epilepsy.base.constants.model_constants import X0_DEF, E_DEF
+from tvb_epilepsy.base.utils.log_error_utils import initialize_logger
+from tvb_epilepsy.base.model.disease_hypothesis import DiseaseHypothesis
 from tvb_epilepsy.service.model_configuration_service import ModelConfigurationService
 from tvb_epilepsy.service.lsa_service import LSAService
 
 
-def start_lsa_run(hypothesis, connectivity_matrix, logger=None):
+LOG = initialize_logger(__name__)
+
+
+def start_lsa_run(hypothesis, model_connectivity, logger=None):
 
     if logger is None:
         logger = initialize_logger(__name__)
 
     logger.info("creating model configuration...")
     model_configuration_service = ModelConfigurationService(hypothesis.number_of_regions)
     model_configuration = model_configuration_service. \
-        configure_model_from_hypothesis(hypothesis, connectivity_matrix)
+        configure_model_from_hypothesis(hypothesis, model_connectivity)
 
     logger.info("running LSA...")
     lsa_service = LSAService(eigen_vectors_number_selection=EIGENVECTORS_NUMBER_SELECTION, eigen_vectors_number=None,
                              weighted_eigenvector_sum=WEIGHTED_EIGENVECTOR_SUM, normalize_propagation_strength=False)
     lsa_hypothesis = lsa_service.run_lsa(hypothesis, model_configuration)
 
-    return model_configuration_service, model_configuration, lsa_service, lsa_hypothesis
\ No newline at end of file
+    return model_configuration_service, model_configuration, lsa_service, lsa_hypothesis
+
+
+def from_head_to_hypotheses(ep_name, data_mode=DATA_MODE, data_folder=os.path.join(DATA_CUSTOM, 'Head'),
```

Comment:
We could move this to a service that creates hypotheses based on (head) epileptogenicity file.

Reply:
This is not a standard procedure since there has to be an interpretation of the epileptogenicity file values into x0 and/or E. This, is really decided on a case by case manner, which is the reason for remaining a helper script for the user...